lib/phlex/fifo.rb

```diff
@@ -1,34 +1,29 @@
 # frozen_string_literal: true
 
 class Phlex::FIFO
-	def initialize(max_bytesize)
+	def initialize(max_size)
```

Comment:
@willcosgrove want to pair on making this auto-configured?

Reply:
Yeah, let's do it!

Reply:
Let’s come back to this in a follow-up.